dm/dm.go

```diff
@@ -190,10 +192,13 @@ func execute() {
 			usage()
 		}
 
-		tUrl := getTypeUrl(args[1])
-		if tUrl == "" {
+		tUrls := getTypeUrl(args[1])
+		var tUrl = ""
+		if len(tUrls) == 0 {
 			// Type is most likely a primitive.
 			tUrl = args[1]
+		} else {
+			tUrl = tUrls[0]
```

Comment:
Added a TODO to handle them properly. I want to do that in a follow on CL since there's other changes necessary for that. Ok?